src/library/export/engineprimeexportjob.cpp

```diff
@@ -230,6 +258,12 @@ void exportMetadata(djinterop::database* pDatabase,
             continue;
         }
 
+        if (!pCue->getPosition().isValid()) {
+            qWarning() << "Hot cue" << hotCueIndex << "exists but is invalid for track"
+                       << pTrack->getId() << "(" << pTrack->getFileInfo().fileName() << ")";
```

Comment:
Making the assumption that it is a warning-worthy situation to have a cue point for a track, but also have an invalid position.